noir_stdlib/src/lib.nr

```diff
@@ -18,5 +18,19 @@ mod compat;
 #[builtin(println)]
 fn println<T>(_input : T) {}
 
+#[oracle(println)]
+unconstrained fn println_oracle<T, A>(_input: T) -> Field {}
+
+#[oracle(println_format)]
+unconstrained fn println_format_oracle<T, A, N>(_msg: T, _args: [A; N]) -> Field {}
+
+unconstrained fn println_new<T>(input: T) {
+    println_oracle(input);
+}
```

Comment:
What is the difference between `println` and `println_new` again? Just that this one is unconstrained? Is the plan to replace `println` with `println_new`? I assume the name is non-final.

Reply:
> I named the first function `println_new` as to not conflict with the existing `println` that uses the old SSA. We could include this as tech debt (#1850) to be removed when we switch to the new SSA or temporarily use a name different than `println` for this function. Either way the name of the Brillig based `println` will be temporary.

From the PR description. The plan is to replace `println` with `println_new`. I had not settled on a final name for `println_new` yet. I also was debating to do what we did for slices and have separate logging modules for the new and old SSA.

Reply:
Why can't we handle the existing `println` function differently depending on if we're in the new or old ssa?

Reply:
I will probably end up doing that. It would be nice to not have `println` changing names

Reply:
I switch which println function to retain in the stdlib in a similar way we do with the slices module for the old SSA